### PR TITLE
`use_logo()` adds appropriate alt-text in README

### LINKED ISSUE #1804: `use_logo()` should add alt text in README html tag
Since the logos are decorative and not informational, the alt-text for them should be `alt=""`

### PR DESCRIPTION
**Pull request by ateucher**
Closes #1804 

* When the logo is not a link to the pkgdown site, alt text is simply "", as the logo is decorative.
* When the logo is also a link to the pkgdown site, the alt text describes the link destination as per accessibility standards (https://www.w3.org/WAI/WCAG21/Techniques/html/H30.html)

### CODE REVIEW

File: R/use_standalone.R [hunk 1]
```diff
@@ -182,7 +182,7 @@ standalone_dependencies <- function(lines, path, error_call = caller_env()) {
   imports <- as_chr_field(yaml$imports)
   imports <- as_version_info(imports, error_call = error_call)
 
-  if (any(na.omit(imports$cmp) != ">=")) {
+  if (any(stats::na.omit(imports$cmp) != ">=")) {
```
Comment (ateucher) [inline R/use_standalone.R:185]: This has nothing to do with this PR, but I've been getting a NOTE every time I run check() locally :)

Comment (jennybc) [reply R/use_standalone.R:185]: Yeah I noticed this reading a GHA log today 👍

File: README.md [hunk 2]
```diff
@@ -61,8 +61,8 @@ library(usethis)
 # Create a new package -------------------------------------------------
 path <- file.path(tempdir(), "mypkg")
 create_package(path)
-#> ✔ Creating '/tmp/Rtmpztuf0Z/mypkg/'
-#> ✔ Setting active project to '/private/tmp/Rtmpztuf0Z/mypkg'
+#> ✔ Creating '/var/folders/5r/wrwbj0sn3m71clq0l1rm4pmm0000gp/T/Rtmpt7SYWI/mypkg/'
+#> ✔ Setting active project to '/private/var/folders/5r/wrwbj0sn3m71clq0l1rm4pmm0000gp/T/Rtmpt7SYWI/mypkg'
```
Comment (jennybc) [inline README.md:65]: This is low-stakes and the ease depends on your git client/workflow. But I often discard these nuisance changes when rebuilding README.

**Review by jennybc — APPROVED** [review]
LGTM

File: README.md (hunk 2 above)
Comment (ateucher) [reply README.md:65]: Yeah, I often do as well...

**Pull request merged**